tests/backend/test_apiv1_organization_stats.py

```diff
@@ -70,3 +96,152 @@ async def test_organization_stats(
 async def test_stats_unknown_organization(administration_backend_sock):
     rep = await organization_stats(administration_backend_sock, organization_id="dummy")
     assert rep == {"status": "not_found"}
+
+
+@pytest.fixture
+async def access_testbed(
+    backend_factory,
+    backend_data_binder_factory,
+    apiv1_backend_sock_factory,
+    organization_factory,
+    local_device_factory,
+):
+
+    async with backend_factory(populated=False) as backend:
+        binder = backend_data_binder_factory(backend)
+
+        org = organization_factory("IFD")
+        device = local_device_factory(
+            org=org,
+            base_device_id="godfrey@d1",
+            base_human_handle="Godfrey Ho <godfrey.ho@ifd.com>",
+            profile=UserProfile.ADMIN,
+        )
+        with freeze_time("2000-01-01"):
+            await binder.bind_organization(org, device)
```

Comment:
Can use `initial_user_manifest_in_v0=True` to avoid creating a workspace for the user manifest
And no need to freeze time here given we don't check the creation date ;-)